### ISSUE TITLE
aarlo.camera_start_recording stopped recording after 30 seconds even when a different duration is set

### ISSUE THREAD

**Issue by chris-chensy** [twrecked/hass-aarlo#245]
I made the recording video request using 'aarlo.camera_start_recording'. I can find the video on the cloud. But it was always around 30s when I set to more than 60s. I guess it is a bug with the service call. And it's the same for 'camera.aarlo_start_recording'.

**Comment by twrecked** [comment]
It looks like I start the underlying system without a specfic timeout and I guess Arlo defaults to 30 seconds. I'll fix that.

**Comment by twrecked** [comment]
This is a little more subtle than I first thought. The problem is when I start recording I don't open a stream to the back end so Arlo stops the stream after 30 seconds. I'll think of a proper fix.

For now, can you try the `camera.record` service? Give it a filename you don't care about.

**Comment by chris-chensy** [comment]
> This is a little more subtle than I first thought. The problem is when I start recording I don't open a stream to the back end so Arlo stops the stream after 30 seconds. I'll think of a proper fix.
> 
> For now, can you try the `camera.record` service? Give it a filename you don't care about.

It is interesting to see that camera.record can storage the video in the specified folder. However, doing so will not save a video on the cloud. I don't use a SD card on my hub so I'm not sure if it will be saved in the storage card.

**Comment by twrecked** [comment]
Crap.

I do have a fix but it's currently in the 0.7 branch, I'll see how easy it is to bring to 0.6.

**Comment by twrecked** [comment]
Fixed in 0.6.18.17 by PR #247 .

To make this work you will need to whitelist you /tmp directory:

```yaml
homeassistant:
  ... other config...
  whitelist_external_dirs:
    - /tmp
```

**Comment by chris-chensy** [comment]
Thanks for fixing it promptly!
